Source/options.h

```diff
@@ -50,6 +50,8 @@ struct GraphicsOptions {
 	bool bColorCycling;
 	/** @brief Enable FPS Limit. */
 	bool bFPSLimit;
+	/** @brief If true, makes durability icon on the HUD gradually turn from gold to red. */
+	bool bDurabilityIconGradual;

```

Comment:
We named the setting "...Durability Warning" in the config, but here it's "Icon".

Additionally, the flag name is weird to read right now.

Suggested rename: `bGradualDurabilityWarning`